tests/Microsoft.Identity.Test.LabInfrastructure/LabAuthenticationHelper.cs

```diff
@@ -72,9 +73,9 @@ public static async Task<AccessToken> GetLabAccessTokenAsync(string authority, s
             {
                 case LabAccessAuthenticationType.ClientCertificate:
                     var clientIdForCertAuth = String.IsNullOrEmpty(clientId) ? LabAccessConfidentialClientId : clientId;
-                    var certThumbprintForLab = String.IsNullOrEmpty(clientId) ? LabAccessThumbPrint : certThumbprint;
+                    // var certThumbprintForLab = String.IsNullOrEmpty(clientId) ? LabAccessThumbPrint : certThumbprint;
```

Comment:
not sure you can remove this. There are actually 2 certs that are being used by our automation.

Reply:
There are two callers to the method. Both are passing String.Empty to certThumbprint. No point in having it.